singleprocess.go

```diff
@@ -161,6 +161,8 @@ func (e *singleprocessEnv) RegisterComponentToStart(_ context.Context, _ string,
 }
 
 func (e *singleprocessEnv) GetComponentsToStart(context.Context, *call.Version) ([]string, *call.Version, error) {
+	// blocks forever to avoid excessive CPU consumption in weavelet.watchComponentsToStart
+	<-make(chan struct{})
```

Comment:
Would it make sense to block on `ctx.Done()` so that this operation can shut down gracefully when the context is cancelled?

```suggestion
func (e *singleprocessEnv) GetComponentsToStart(ctx context.Context, _*call.Version) ([]string, *call.Version, error) {
	// Block forever since there's not going to be anything to start.
	<-ctx.Done()
```

Reply:
Ah yes, this is indeed better.